resources/community/devhubOnMinishift.md

```diff
@@ -0,0 +1,91 @@
+ #Setup Docker in localhost
+####In case you want to run Devhub on local Minishift:
```

Comment:
@w8896699 Can you correct the headings ?

`####In Case` -> `#### In Case` 

Also using `#### <text>` is equivalant to an `<h4>`. I think you may want to start off with using h2's or h3s as top level subheadings before you go down to h4. 

In addition, the title '__In case you want to run Devhub on local Minishift__' is way to specific to running 'Devhub'. This should be for anyone that wants to run openshift locally that would like to leverage bcgov images.